docs/current_docs/index.mdx

```diff
@@ -26,7 +26,7 @@ Dagger is an open-source runtime for composable workflows. It's perfect for syst
 
 - **Interactive Terminal:** Directly interact with your workflow or agents in real-time through your terminal. Prototype, test, debug, and ship even faster.
 
-<img src="img/spider-robot-1.svg" width="15%"></img>
```

Comment:
If this image is not in use any longer, should we delete it?

Reply:
updated it to good one.